src/util/HttpServer/ContentEncodingHelper.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include "./beast.h"
+
+namespace ad_utility::content_encoding {
+
+enum class CompressionMethod { NONE, DEFLATE, GZIP };
+
+namespace detail {
+
+constexpr CompressionMethod getCompressionMethodAcceptEncodingHeader(

```

Comment:
`getCompressionMethodFromAcceptEncodingHeader`, then you can really drop the comment.